test/utility/fragments.ts

```diff
@@ -1158,9 +1185,15 @@ export const location = gql`
         ...fieldRegion
       }
     }
+    defaultMarketingRegion {
```

Comment:
Since including this in the requested location, it would be good to actually confirm functionality of this new field in the tests.